src/Confluent.Kafka/Serialization/NullSerializer.cs

```diff
@@ -14,22 +14,33 @@
 //
 // Refer to LICENSE for more information.
 
+using System;
+using System.Collections.Generic;
+
+
 namespace Confluent.Kafka.Serialization
 {
     /// <summary>
     ///     A dummy serializer for use with values that must be null (the <see cref="Null"/> class cannot be instantiated).
```

Comment:
It's never used - it's only there to make the type system happy. So it is different to the other serializers, so I take the position that labelling it as dummy is somewhat meaningful..